tests/rules/test_brew_install.py

```diff
@@ -1,45 +1,36 @@
 import pytest
 from thefuck.rules.brew_install import match, get_new_command
-from thefuck.rules.brew_install import _get_formulas
 from thefuck.types import Command
 
 
 @pytest.fixture
 def brew_no_available_formula():
-    return '''Error: No available formula for elsticsearch '''
+    return '''Warning: No available formula with the name "gitt". Did you mean git, gitg or gist?'''
```

Comment:
I say we miss some possibly good extra suggestions here. How about extending the rule to suggesting those other too? e.g. for `brew install elsticsearch`, The Fuck would first recommend `brew install elasticsearch` then `brew install elasticsearch@6`.

Reply:
Yes, I like the idea. I will look up some other rules to understand how the multi suggestion works.